source/Grid/Grid.js

```diff
@@ -284,9 +265,9 @@ export default class Grid extends PureComponent {
     estimatedRowSize: 30,
     getScrollbarSize: scrollbarSize,
     noContentRenderer: () => null,
```

Comment:
I think this should be annotated explicitly as `() => React.Element<*> | null` — my guess is flow messes this up with `$DIff<Props, DefaultProps>` somehow.